### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by ghost** [twitter/typeahead.js#1777]
How does typeahead.js support the mobile end, after I introduce it in the project, it will fail on the mobile end

**Comment by simplenotezy** [comment]
No, it will not. Also, have a look at https://github.com/corejavascript/typeahead.js/ which is a somewhat maintained fork of this library

**Comment by prabasak** [comment]
I have a project which use this plugin and fails in mobile device, iPad in this case. I am not sure will it work if I target my plugin to the 'corejs-typeahead' instead. How much of a code change will be required. Please help me out on this.
